test/Lachain.CoreTest/RPC/HTTP/Web3/AccountServiceWeb3Test.cs

```diff
@@ -24,6 +24,10 @@
 using Lachain.Crypto.Misc;
 using Lachain.Core.Blockchain.Pool;
 using Nethereum.Signer;
+using Transaction = Lachain.Proto.Transaction;
```

Comment:
looks like this is not being used in the code; if that is the case we can remove this